src/deepsparse/server/config.py

```diff
@@ -49,6 +49,63 @@ class ImageSizesConfig(BaseModel):
     )
 
 
+class SystemLoggingGroup(BaseModel):
+    """
+    Holds the configuration for a single system logging group.
+    """
+
+    enable: bool = Field(
+        default=False, description="Whether to enable the system logging group"
+    )
+
+    target_loggers: Optional[List[str]] = Field(
+        default=None,
+        description="The list of target loggers to log to. "
+        "If None, logs to all the available loggers",
+    )
+
+
+class SystemLoggingConfig(BaseModel):
+    """
+    Holds the configuration for the system logging
+    """
+
+    # Global System Logging
+    enable: Optional[bool] = Field(
+        default=True, description="Whether to enable system logging"
+    )
+    target_loggers: Optional[List[str]] = Field(
+        default=None,
+        description="The list of loggers to log to. If None, "
+        "logs to all the available loggers.Will "
+        "be overriden by the target_loggers specified "
+        "in the individual system logging groups.",
+    )
+
+    # System Logging Groups
+    resource_utilization: Optional[SystemLoggingGroup] = Field(
+        default=None,
+        description="The configuration group for the resource "
+        "utilization system logging group. For details "
+        "refer to the DeepSparse server system logging "
+        "documentation. By default this group is disabled.",
+    )
+    prediction_latency: SystemLoggingGroup = Field(
+        default=None,
+        description="The configuration group for the prediction latency "
+        "system logging group. For details "
+        "refer to the DeepSparse server system logging "
+        "documentation. By default this group is disabled.",
+    )
```

Comment:
Could this be handled on the pipeline logger config level?

Reply:
latency was agreed to be a "system" level log, so should fall in this config. afair there isn't a way to disable these right now

Reply:
Well, depends what you mean by disable. 
We are unable to disable prediction latency generation (this happens on the pipeline level), but we are able to disable it on the logger level (just refrain from creating the FunctionLogger with appropriate identifier)